applications/nrf_desktop/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ cmake_minimum_required(VERSION 3.20.0)
 set(APPLICATION_CONFIG_DIR "${CMAKE_CURRENT_LIST_DIR}/configuration/\${NORMALIZED_BOARD_TARGET}")
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
-project("nRF52 Desktop"
+project("nRF Desktop"
```

Comment:
Regarding commit message:

```
Removes "52" infix from the nRF Desktop devices names. This change
breaks backwards compatibility. New peripherals won't pair with old dongles
and the other way around. Also aligned HID configurator script.
 ```

`Also DFU through dongle from the old naming scheme to a new one may fail in some cases when the first device that is updated is dongle.` -> Should not be problematic unless an user explicitly removes bonds (reconnections would work fine; `bt_keys` would not be dropped by the Bluetooth stack if we only disable the legacy pairing - checked today)